README.md

```diff
@@ -395,6 +397,53 @@ or send an email to [info@pyrgg.ir](mailto:info@pyrgg.ir "info@pyrgg.ir").
        node_n,node_2,weight_n 
 	```
 
+- GML(.gml)
+	```
+	   graph
+	   [
+         multigraph 0
+         directed  0
+         node
+         [
+          id 1
+          label "Node 1"
+         ]
+         node
+         [
+          id 2
+          label "Node 2"
+         ]
+         .
+         .
+         .
+         [
```

Comment:
It'd better if we add `node` here, like:
```
- [
+ node
+ [
```

Reply:
Thanks, fixed in https://github.com/sepandhaghighi/pyrgg/pull/46/commits/19c53dc883ca6b2e09474c8ed305a0449586589e